bin/subzero/runtime/Cargo.toml

```diff
@@ -43,6 +43,7 @@ pallet-membership = { git = "https://github.com/paritytech/substrate", default-f
 pallet-multisig = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.36" }
 pallet-preimage = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.36" }
 pallet-proxy = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.36" }
+pallet-randomness-collective-flip = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.36" }
```

Comment:
According to [substrate docs](https://docs.substrate.io/build/randomness/#generating-randomness), this implementation of randomness is not suggested for production.

Reply:
yes there is no alternative to my knowledge, so i have added it as well